### ISSUE TITLE
Leaderboard names colours reflecting mystics/scythes/scouts/moderators/non-promoted.

### ISSUE THREAD

**Issue by Nseraf** [crazyman4865/EWDLC#1]
Make the leaderboard name colours reflecting rank

non promoted
if scout show scout 
if scout + mod show scout
if scythe show scythe 
if scythe and mod show scythe
if mod show mod
if mystic show mystic 
if mystic and mod show mystic
if admin show admin 

Yeah ik a mystic has scout and scythe but it shows mystic in profile so going with that.

**Comment by RandomGuyJCI** [comment]
How about mentor?

**Comment by Nseraf** [comment]
lol yeah mentor as well, i just gave some examples.

**Comment by crazyman4865** [comment]
Should the role that the player has chosen to be displayed as in chat be taken into account?

**Comment by Nseraf** [comment]
sure but then it'll display me as silver? lol

**Comment by annkri1** [comment]
think it should be the highest role that is displayed so we are not able to hide what we are scout- scythe- mystic - admin not sure what to do with mentor and mod perhaps if possible you can ad a small icon for each of them?

**Comment by Nseraf** [comment]
I agree, I think it should go scouts > scythe > mystic > moderator > unique colour > admin (although considering that most mystics are also mods I'd either let mystics + mods decide or have mystic override mod).  Can't be a mentor without also being at least a scout.

**Comment by annkri1** [comment]
like i see it mod and scout are the same rank since you can get both from nonpromoted, and i would rank mentor between scythe and mystic since as a general rule imo you need to be a scythe to know enogh for a good mentor (and then you can correct mergers right away), and many of the mystics have all roles so that is the highest order, but it is really difficult to rank mod and mentor so why i would like a small badge for it like the flagg and ew logo for hq. as to unique colour i know that is something to be proud of but the way i am thinking we should be designing it so it could work like a guide of what rank we have to players that do not know us. and if unique color first is there, it would not be long before someone would want to have topscoore colour also, and in chat that is the most confusing colour. and by the same reasoning the chosen chatcolour should not have anything to say  on leaderboard colour

**Comment by ChrisRaven** [comment]
I like the idea @annkri1 mentioned with small icons. For the order, what @Nseraf suggested seems reasonable. Only, instead of icons, I'll add e.g. a red dot after player's name, if they is a mod and e.g. a blue dot, if the player is a mentor (if player has both roles, then, ofc, 2 dots).

**Comment by Nseraf** [comment]
I just wanted multi-coloured leaderboard(s) xD :P (names).

**Comment by ChrisRaven** [comment]
Lol, yeah, colors are cool xD However, if that feature make its way to the EW itself, it might be useful for the normal players to easily recognize, which players are e.g. mentors and can watch them.

**Comment by Nseraf** [comment]
That is quite true. :)

**Comment by Nseraf** [comment]
Okay, how does this sound? Leaderboard displays whatever (rank only not topscore) chat team player has on, and next to their name a miniature mentor and/or mod badge icons, really really small so it doesn't upset the leaderboard name placements (leaving spaces etc below/above to accommodate the icon(s).  Something like this:
![leader test](https://user-images.githubusercontent.com/33002317/32172561-b6ee01a6-bd84-11e7-869f-663f143ae7e6.jpg)
